### PR TITLE
fix(karma-test-runner): Remove flags from requested packages

### PR DESCRIPTION
**Pull request by NataliaMarkaki**

### CODE REVIEW

File: karma-test-runner/src/index.js [hunk 1]
```diff
@@ -21,7 +22,7 @@ const utsOnly = runOnlyUTs(args);
 // If true keep browser open after test run.
 const devMode = args.dev || false;
 // Packages to test, if the user specifies none all packages will be tested.
-const packagesToTest = args._;
+const packagesToTest = getRequestedPackages(args._);
```
Comment (briandipalma) [inline karma-test-runner/src/index.js]: This is a bit confusing, we are using `minimist` to parse the CLI args. Why do we need to write our own parsing code? Can we not just minimist to parse these tail arguments?

Comment (NataliaMarkaki) [inline karma-test-runner/src/index.js]: It didn't work properly and all of our args where being stored in the array.

**Comment by thecapdan** [comment]
I can confirm this is now working on windows with and without the ':serve'

File: karma-test-runner/src/index.js (hunk 1 above)
Comment (briandipalma) [inline karma-test-runner/src/index.js]: I think yes, it would be good to understand why minimist isn't working.

**Comment by briandipalma** [comment]
Given the test runner has been rewritten these changes are no longer applicable so closing PR.

**Pull request closed without merging**